src/js/components/Button/README.md

```diff
@@ -160,19 +160,21 @@ boolean
 **hoverIndicator**
 
 The hover indicator to apply when the user is mousing over the
-button. An object can be also be specified for color index support:
-{background: 'neutral-2'}. This prop is meant to be used only
-with plain Buttons.
+button. HoverIndicator can accept a boolean, string, and object
+Where the value of true or "background" applies theme.global.hover.background;
+a string specifying a color (e.g. '#999999'; 'neutral-1') Or, a 'url()' for an image;
+an object specifying color index support: (e.g.
+{color: 'neutral-2'}, {"dark": "light-1", "light": "dark-2"}).
```

Comment:
This is not specific for hoverIndicator, so I don't think it is the right place.

Reply:
Agree we can move this to one wiki page will start working on that for users!